misc-scripts/generate_stable_ids.pl

```diff
@@ -171,7 +171,7 @@ sub get_highest_stable_id {
 
   # Get highest stable ID from the relevant table.
 
-  my $sth = $dbi->prepare("SELECT MAX(stable_id) FROM $type");
+  my $sth = $dbi->prepare("SELECT MAX(stable_id) FROM $type WHERE stable_id LIKE 'ENS%'");
```

Comment:
This is probably the best compromise for now. This script isn't set up to handle weird cases like LRGs or species that don't follow the typical schema (e.g. saccharomyces_cerevisiae). Anyone working with those stable ids needs a different script.

Although at the moment there aren't any LRGs in the human gene_archive, for consistency, you should make this change in `get_max_stable_id_from_gene_archive` (at line 148) as well.

Reply:
Thanks Brandon, I've updated the SQL for the gene archive query.